homeworkWeekend10/src/ba/bitcamp/SQL/task01/SQLConnection.java

```diff
@@ -0,0 +1,104 @@
+package ba.bitcamp.SQL.task01;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * Represents the class that contains method that enables loading JDBC library,
+ * and method that enables creation and connection with the database. Also it
+ * contains methods which enables inputing new complaints into database, and
+ * viewing all inputed complaints.
+ * 
+ * @author Gordan
+ *
+ */
+public class SQLConnection {
+
+	// Declaring variable that enables connection with database.
+	private static Connection conn = null;
+
+	/**
+	 * Loads JDBC library.
+	 */
+	private static void loadLibrary() {
+		try {
+			Class.forName("org.sqlite.JDBC");
```

Comment:
Driver name, connection string, etc. should be declared as static members of this class. Having 'magic' strings present trough the class makes it really hard to maintain.